jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -63,7 +64,7 @@
      *
      * @param tarArchiveEntry the {@link TarArchiveEntry}
```

Comment:
Huh, doesn't it complain that you don't have `@throw` here?

Reply:
I can for sure add one there too

Reply:
done